docs/examples.md

```diff
@@ -23,7 +23,7 @@ Tests repository demonstrate usage of
 * PageObjects
 * Cucumber syntax
 
-## [Basic Examples](https://github.com/codecept-js/CodeceptJS/tree/master/examples)
+## [Basic Examples](https://github.com/Codeception/CodeceptJS/tree/master/examples)
```

Comment:
should it be https://github.com/codeceptjs/CodeceptJS/tree/master/examples?

Reply:
you are redirected to the proper place, so I guess, this should be fine, thou!